File: UKParliament.CodeTest.Data/Person.cs
namespace UKParliament.CodeTest.Data;

public class Person
{
    public int Id { get; set; }

    public required string FirstName { get; set; } = string.Empty;

    public required string LastName { get; set; } = string.Empty;

    public required DateTime DateOfBirth { get; set; }

    public required int DepartmentId { get; set; }

    public Department? Department { get; set; }

    public  string Email { get; set; } = string.Empty;

    public string PhoneNumber { get; set; } = string.Empty;

    public string ProfileImageUrl { get; set; } = string.Empty;

    public bool IsActive  { get; set; }
}

File: UKParliament.CodeTest.Web/ClientApp/src/models/person.ts
export interface Person {
    id?: number | null;
    firstName: string;
    lastName: string;
    dateOfBirth: string;
    departmentId: number;
    departmentName?: string;
    email: string;
    phoneNumber: string;
    profileImageUrl?: string | null;
    isActive?: boolean;
}